.github/CODEOWNERS

```diff
@@ -433,7 +433,7 @@
 # ServiceOwners:                                                   @adriankjohnson
 
 # ServiceLabel: %Database Watcher
```

Comment:
@ronniegeraghty:  Can you please offer thoughts on this label and whether it should be centrally managed?